templates/layout/_options_theme.html.twig

```diff
@@ -5,36 +5,36 @@
        title="{{ 'kbin' }} {{ 'theme'|trans }}"
        aria-label="{{ 'kbin' }} {{ 'theme'|trans }}"
        rel="nofollow">
-        <div class="{{ html_classes('theme', 'kbin', constant('App\\Controller\\User\\ThemeSettingsController::KBIN') == theme ? 'active' : '' ) }}"></div>
```

Comment:
I mentioned in the commit but these active classes didn't seem to do anything, the outline to highlight the theme is selected comes from `assets/styles/components/_sidebar.scss` which is solely based off css selectors like `.theme--dark .theme.dark`, as that is enough to tell which theme is active. There weren't any styles added by the `.active` as far as I could see